src/features/query-bar/constants.js

```diff
@@ -20,47 +19,35 @@ export const FLOAT: 'float' = 'float';
 export const ENUM: 'enum' = 'enum';
 export const DATE: 'date' = 'date';
 
+export const IS: '=' = '=';
+export const IS_NOT: '!=' = '!=';
+export const GREATER_THAN: '>' = '>';
+export const LESS_THAN: '<' = '<';
+
+export const IS_OPERATOR = {
```

Comment:
Can I suggest this naming convention?

```export const OPERATORS = {
  IS: { },
  GREATER_TAN: { },
};

console.log(OPERATORS.IS);
```

Also let's rename IS to EQUALS. "IS" looks too much like "one S".